eGovPDX/portlandor
# [POWR-1980] Watch speed improvements

[POWR-1980](https://portland.atlassian.net/browse/POWR-1980) 

## Test plan

  * Run `lando yarn install`
  * Run `lando yarn run start`
  * Make a change to a twig template. Watch should rebuild the necessary demos more quickly.
  * Make a change to a sass file. Watch you rebuild the necessary css.
  * Make sure that tokens are properly building out css when a token change is made.
